gptme/cli.py

```diff
@@ -38,7 +38,9 @@
 
 script_path = Path(os.path.realpath(__file__))
 commands_help = "\n".join(_gen_help(incl_langtags=False))
-available_tool_names = ", ".join([tool.name for tool in all_tools if tool.available])
+available_tool_names = ", ".join(
+    [tool.name for tool in ToolSpec.get_tools().values() if tool.available]
+)
```

Comment:
This should perhaps not be loaded at module-init time anymore, since tools may be registered after init.

Reply:
It's only for `--help`, so its fine.